packages/zapp/console/src/components/Entities/EntityInputs.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import CheckIcon from '@material-ui/icons/Check';
+import { useLaunchPlans } from 'components/hooks/useLaunchPlans';
+import { formatType, getInputDefintionForLiteralType } from 'components/Launch/LaunchForm/utils';
+import { FilterOperationName } from 'models/AdminEntity/types';
+import { ResourceIdentifier } from 'models/Common/types';
+import { LaunchPlanClosure, LaunchPlanSpec } from 'models/Launch/types';
+import * as React from 'react';
+import t from './strings';
+import { transformLiterals } from '../Literals/helpers';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  header: {
+    marginBottom: theme.spacing(1),
+  },
+  divider: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    marginBottom: theme.spacing(1),
+  },
+  rowContainer: {
+    display: 'flex',
+    marginTop: theme.spacing(3),
+  },
+  firstColumnContainer: {
+    width: '60%',
+    marginRight: theme.spacing(3),
+  },
+  secondColumnContainer: {
+    width: '40%',
+  },
+  configs: {
+    listStyleType: 'none',
+    paddingInlineStart: 0,
+  },
+  config: {
+    display: 'flex',
+  },
+  configName: {
+    color: theme.palette.grey[400],
+    fontSize: '14px',
+    marginRight: theme.spacing(2),
+    minWidth: '95px',
+  },
+  configValue: {
+    color: '#333',
+    fontSize: '14px',
```

Comment:
Font Sized for this two should come from  body1 vairant of typography`<Typography  variant="body1">`,
as per figma
<img width="1025" alt="Screen Shot 2022-06-26 at 7 03 37 PM" src="https://user-images.githubusercontent.com/55718143/175825733-1220fcd4-bf33-4c1b-9f03-ce3f0e3e33ab.png">
Using proper typography will allow us in future to change font specific per type in one place, instead of going through each item one by one
<img width="1025" alt="Screen Shot 2022-06-26 at 7 03 37 PM" src="https://user-images.githubusercontent.com/55718143/175825756-52f3a948-3515-44ed-8ef8-3f7c44a8d1d7.png">

Reply:
fixed